### PR TITLE
[add] check can run 'lsb_release'

### PR DESCRIPTION
**Pull request by kazukazuinaina**
#### Requirements (please check off with 'x')

- [x] I have read the [Contributing Guidelines](https://github.com/ryanoasis/vim-devicons/blob/master/contributing.md)
- [x] I have read or at least glanced at the [FAQ](https://github.com/ryanoasis/vim-devicons#faq--troubleshooting)
- [x] I have read or at least glanced at the [Wiki](https://github.com/ryanoasis/vim-devicons/wiki)

#### What does this Pull Request (PR) do?

check to can run `lsb_release -i`

#### Any background context you can provide?

#310 's PR

#### Screenshots (if appropriate or helpful)

Could you check this Pull Request?

### CODE REVIEW

File: plugin/webdevicons.vim [hunk 1]
```diff
@@ -96,23 +96,24 @@ function s:getDistro()
...
+    elseif s:lsb =~# 'Dock'
+      let s:distro = ''
+    else
+      let s:distro = ''
+    endif
+    return s:distro
   endif
-
-  return s:distro
+  return
```
Comment (tsuyoshicho) [inline plugin/webdevicons.vim]: This function contract should need return s:distro.

If non executable, set default value.
And at last, return s:distro.

File: plugin/webdevicons.vim [hunk 2]
```diff
@@ -96,23 +96,24 @@ function s:getDistro()
...
-    let s:distro = ''
-  elseif s:lsb =~# 'Cent'
-    let s:distro = ''
-  elseif s:lsb =~# 'Debian'
-    let s:distro = ''
-  elseif s:lsb =~# 'Dock'
-    let s:distro = ''
-  else
-    let s:distro = ''
+  if executable('lsb_release -i')
```
Comment (tsuyoshicho) [inline plugin/webdevicons.vim]: Executable need only exec target file.
Remove `-i`

Comment (kazukazuinaina) [reply plugin/webdevicons.vim]: @tsuyoshicho
Thanks for code review.
I fixed

File: plugin/webdevicons.vim [hunk 3]
```diff
@@ -96,22 +96,25 @@ function s:getDistro()
...
+      let s:distro = ''
+    elseif s:lsb =~# 'Dock'
+      let s:distro = ''
+    else
+      let s:distro = ''
+    endif
+    return s:distro
   endif
 
+  let s:distro = ''
```
Comment (tsuyoshicho) [inline plugin/webdevicons.vim]:
```suggestion
  let s:distro = ''
```

Unknown distro shall have default icon at Tux, I think it.

Comment (kazukazuinaina) [reply plugin/webdevicons.vim]: Thanks, I fixed.

**Comment by tsuyoshicho** [comment]
Thanks, and good work!

**Comment by kazukazuinaina** [comment]
@ryanoasis 
Should I need squash?

**Comment by ryanoasis** [comment]
So this fixes issue if can't run the command or it doesn't exist right?

I haven't been able to test the PRs as thoroughly but maybe we need some automated unit tests for some things.


No need to squash this seems fine.

If you can just confirm my question then I'll happily merge this :smile:

**Comment by kazukazuinaina** [comment]
> So this fixes issue if can't run the command or it doesn't exist right?

Yes, that's right.

**Comment by kazukazuinaina** [comment]
@ryanoasis 
Thanks for checking.

**Pull request merged**